.idea/workspace.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectId" id="1at6GbNNng2Brbsdcw2MMIMFR5A" />
```

Comment:
Can you add .idea/ to .gitignore?

Reply:
Or git rm as I can see it's already there.

Reply:
Sure, will do.